### PR TITLE
Assert V2

### PR DESCRIPTION
**Pull request by tyler-whitman**
Reworks the assert flow to no longer require hotspot pairing. This introduces the Assert V2 transaction to the app.

This is still pending the API change for https://github.com/helium/blockchain-http/issues/208 and a new release of Helium JS.

### CODE REVIEW

File: src/utils/polyfill.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Platform } from 'react-native'
+
+// noinspection JSConstantReassignment
```
Comment (matthewcarlreetz) [inline src/utils/polyfill.js:3]: what's this here for?

File: src/components/HotspotConfigurationPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    onGainUpdated(antenna.gain)
+    setGain(
+      antenna.gain.toLocaleString(locale, {
+        maximumFractionDigits: 1,
+        minimumFractionDigits: 1,
+      }),
+    )
+  }
+
+  const showElevationInfo = () =>
```
Comment (matthewcarlreetz) [inline src/components/HotspotConfigurationPicker.tsx:84]: Is this placeholder?

File: src/components/HotspotConfigurationPicker.tsx [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+      borderRadius="m"
+      marginVertical="l"
+      borderWidth={outline ? 1 : 0}
+      borderColor="grayLight"
+    >
+      <HeliumActionSheet
+        title={t('antennas.onboarding.select')}
+        prefixVariant="regular"
+        prefixFontSize={14}
+        data={Object.values(Antennas).map((a) => ({
```
Comment (matthewcarlreetz) [inline src/components/HotspotConfigurationPicker.tsx:161]: Wouldn't hurt to move this data out to a `useMemo`

File: src/features/hotspots/settings/ReassertAddressSearch.tsx [hunk 4]
```diff
@@ -99,7 +101,7 @@ const ReassertAddressSearch = ({ onSelectPlace }: Props) => {
   )
 
   return (
-    <Box height={hp(75)} padding="none" paddingTop="lx">
+    <Box height={600} padding="none" paddingTop="lx">
```
Comment (matthewcarlreetz) [inline src/features/hotspots/settings/ReassertAddressSearch.tsx]: Is this too tall for some phones?

**Review by matthewcarlreetz — COMMENTED** [review]
Code is looking good. Excited to try it out tonight.

**Review by matthewcarlreetz — APPROVED** [review]
Worked great!

File: src/utils/polyfill.js (hunk 1 above)
Comment (tyler-whitman) [reply src/utils/polyfill.js:3]: This is for H3, it needs a shim to work but it doesnt actually use the document so it works great. I added that noinspect to ignore some lint error.

File: src/components/HotspotConfigurationPicker.tsx (hunk 2 above)
Comment (tyler-whitman) [reply src/components/HotspotConfigurationPicker.tsx:86]: Its the alerts that pop up when you hit the i icons near the elevation and gain text. Shows an alert with some info.

File: src/components/HotspotConfigurationPicker.tsx (hunk 3 above)
Comment (tyler-whitman) [reply src/components/HotspotConfigurationPicker.tsx:170]: Done

File: src/features/hotspots/settings/ReassertAddressSearch.tsx (hunk 4 above)
Comment (tyler-whitman) [reply src/features/hotspots/settings/ReassertAddressSearch.tsx]: I think it should be ok, when we use the hp I notice things sometimes differ but fixed pixel is supposed to be density independent so it should always be the same height on any device. Let me try it on a small phone though

Comment (tyler-whitman) [reply src/features/hotspots/settings/ReassertAddressSearch.tsx]: It is the same height but its taller than the device on the SE lol. Ill make it smaller for small devices

**Pull request merged**